hnn_core/batch_simulate.py

```diff
@@ -70,10 +73,22 @@ def __init__(self, set_params, net_name='jones', tstop=170,
         summary_func : callable, optional
```

Comment:
Also is this is `None`, and `clear_cache=True`, then there should be no return from `batch_simulate.run()`